bundle-ml/src/main/scala/ml/combust/bundle/serializer/FileUtil.scala

```diff
@@ -29,7 +29,11 @@ case class FileUtil() {
       if(entry.isDirectory) {
         new File(dest, entry.getName).mkdirs()
       } else {
+        val destCanonical = dest.getCanonicalPath()
         val filePath = new File(dest, entry.getName)
+        if(!filePath.getCanonicalPath().startsWith(destCanonical + File.separator)){
+          throw new Exception("Entry is outside of the target dir: " + e.getName())
```

Comment:
test suite showing this as not compiling.  Think it needs to be `entry.getName()`?

Reply:
Sure